ivi-layermanagement-examples/LayerManagerControl/src/print.cpp

```diff
@@ -94,6 +94,8 @@ void printScreenProperties(unsigned int screenid, const char* prefix)
         return;
     }
 
+    cout << prefix << "- connector name:       " << screenProperties.connectorName << "\n";
+
     cout << prefix << "- resolution:           x=" << screenProperties.screenWidth << ", y="
```

Comment:
Tested on rcar starterkit m3. Connector name can be obtained by means of LayerManagerControl get scene.